driver-examples/osgi/src/main/java/com/datastax/driver/osgi/impl/Activator.java

```diff
@@ -17,10 +17,13 @@
 
 import java.util.Hashtable;
 
+import com.sun.org.apache.xpath.internal.operations.Bool;
+import io.netty.channel.socket.SocketChannel;
```

Comment:
I would like to avoid having the osgi project's pom depending on both the shaded and non-shaded artifacts, for the very same reasons you mentioned. 
Furthermore my understanding is that the situation where a client would use the shaded version while still wanting to customize Netty will not be supported, so I simply ignored this configuration. @olim7t What's your opinion here? Should we explicitly support shaded Netty + Netty customization?